usaspending_api/common/query_with_filters.py

```diff
@@ -130,7 +137,7 @@ def generate_elasticsearch_query(cls, filter_values: str, query_type: QueryType,
             query += r"\*"
         else:
             query += "*"
```

Comment:
The `if/else` block can be dropped, because `multi_match` doesn't support wildcards in the query text and will treat them as normal characters. So a search for `test` is being turned into a search for `test*`.